_guidelines-v4/05-mensural.md

```diff
@@ -10,8 +10,8 @@ sectionid: "mensural"
 
-There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main particularities is that the actual duration of a note is not given only by its symbol but also by position and the context in which the symbol appears. The general context is given by one of the 16 mensural *species* provide four levels of division: *modus major*, *modus minor*, *tempus* and *prolatio*. Depending on the context, certain rules must be applied in order to determine the duration of a note. In these cases, encoding both the sign and its actual duration is highy desirable.
+There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests in order to appropriately encode mensural durations. One of the main features is that the actual duration of a note is specified not only by its symbol but also by its position and the context in which the symbol appears. The general context is given by one of the 16 mensural *species* provided by the four levels of division: *modus major*, *modus minor*, *tempus*, and *prolatio*. Depending on the context, certain rules must be applied to determine the duration of a note. Therefore, encoding information on both the sign and its actual duration is highly desirable.
 
-Another particularity of mensural notation is the use of proportions that are indicated by numeric proportions or by specific mensuration signs. The proportions indicate that the durations have to be modified accordingly and they can be combined in a very complex manner. Over time, proportions and mensuration signs were simplified and became time signatures in CMN. The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module.
+Another feature of mensural notation is the use of proportions that are indicated by numeric proportions or by specific mensuration signs. Proportions indicate that the durations have to be modified accordingly. Proportions can be combined in a very complex manner. Over time, proportions and mensuration signs were simplified and became time signatures in CMN. The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module.
```

Comment:
You can't use 'accordingly' without giving the specifics of the modifications. "The pressure gauge should read Xpsi, so the valve should be modified accordingly."

Reply:
Thanks! I am modifying it to (see text bolded text):

> Another feature of mensural notation is the use of proportions that are indicated by numeric ratios or by specific mensuration signs. **Proportions indicate that the durations have to be modified according to the given ratios.** Proportions can be combined in a very complex manner. Over time, proportions and mensuration signs were simplified and became time signatures in CMN. The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module.